preprocessing/pitch_harvest.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Jan 30 09:34:53 2020
+
+@author: linhailan1
+
+Extract beats and pitches, using harvest algorithm
+"""
+
+
+import argparse
+import librosa
+import os
+import numpy as np
+import pyworld as pw
+
+parser = argparse.ArgumentParser()
+parser.add_argument("datadir", type=str, help="data directory")
+parser.add_argument("outdir", type=str, help="output directory")
+#parser.add_argument("model", type=str, help="model type")
+args = parser.parse_args()
+'''
+if args.model == 'HMM':
+    frame_length = 25/1000
+    frame_shift = 10/1000
+elif args.model == 'TDNN':
```

Comment:
as you may find, we already support pitch extraction using harvest. please remove it